radio/src/gui/128x64/model_setup.cpp

```diff
@@ -1059,7 +1059,7 @@ void menuModelSetup(event_t event)
       case ITEM_MODEL_EXTERNAL_MODULE_FAILSAFE: {
         uint8_t moduleIdx = CURRENT_MODULE_EDITED(k);
         ModuleData &moduleData = g_model.moduleData[moduleIdx];
-        if (IS_MODULE_PXX(moduleIdx)) {
+        if (IS_MODULE_PXX(moduleIdx) || IS_MODULE_MULTIMODULE(moduleIdx)) {
```

Comment:
I think we should get rid of this if completely. Other ui (horus, x9) also don't have an if here